api/app/App.js

```diff
@@ -319,4 +320,26 @@ module.exports = function() {
             console.log(e);
         }
     });
+
+    //zapier authentication
+    global.app.get('/zapier/app/:appId/:apiKey', function (req, res) {
+        var appId = req.params.appId;
+        var apiKey = req.params.apiKey;
+        var project = null;
+        global.appService.getAllProjects().then(function (projects) {
+            projects.forEach(function (element) {
```

Comment:
why querying all projects and then filtering out one ? just write a query to find the required project ..